test/core/componentTests.ts

```diff
@@ -244,7 +244,7 @@ describe("Component behavior", () => {
     svg.remove();
   });
 
-  it("hitboxes are created iff there are registered interactions", () => {
+  it("hitboxes are created iff there are registered interactions (that require hitboxes)", () => {
```

Comment:
Well, if they didn't require hitboxes, then one isn't created. The parens help designate exactly what functionality is being tested.